docs/src/examples/elements/Label/Types/LabelExampleLabel.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { Icon, Label } from 'semantic-ui-react'
 
-const LabelExampleBasic = () => (
+const LabelExampleLabel = () => (
   <Label>
     <Icon name='mail' /> 23
   </Label>
 )
 
-export default LabelExampleBasic
+export default LabelExampleLabel
```

Comment:
I had to change this because it turns out there's another component with the [exact same name](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/src/examples/elements/Label/Variations/LabelExampleBasic.js) so sometimes https://react.semantic-ui.com/maximize/label-example-basic/ would render one component first, other times the other one.

Reply:
I think there is an eslint rule we could add to catch issues like this. I'd have to go look and see what we used in another project but it was helpful. Especially in tests.